src/applications/static-pages/ask-va/helpers/index.js

```diff
@@ -0,0 +1,32 @@
+// Relative imports.
+import environment from 'platform/utilities/environment';
+
+export const deriveDefaultURL = () => {
+  // Production environments.
+  if (environment.isProduction()) {
+    return 'https://ava.va.gov/';
```

Comment:
So this URL is not live yet?

Reply:
That's correct, yupp! Not until October I think?

Reply:
DERP, I just saw your comment below, disregard.

Reply:
haha yupp the react widget is hidden on production currently! 🙂